README.md

```
@@ -69,7 +76,7 @@ class ReadmeTest {
 val mokksy = Mokksy()
 ```
 
-2. Create an http client using MokksyServer's as baseUrl:
+2. Configure http client using Mokksy server's as baseUrl in your application:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)